hooks/stages/allbidresponse.go

```diff
@@ -0,0 +1,16 @@
+package stages
+
+import (
+	"context"
+
+	"github.com/prebid/prebid-server/hooks/invocation"
+)
+
+type AllBidResponsesHook interface {
+	Code() string
```

Comment:
What is the purpose of `.Code()`?

Reply:
We assume the module could provide several implementations of the same hook interface. So admin when configuring the hook execution plan can specify the code of the hook he wants to use.
So we added hook code to distinguish different hooks of the same hook interface (similar to PBS-Java).
But, you are right, we can remove the `Code` method from the Hook interface, we can use map for this. You will have more context after we add PRs for module loading and hook invocation tasks.